.travis.yml

```diff
@@ -16,6 +16,7 @@ sources:
 
 # What to install before the build
 before_install:
+  - cmake --version
```

Comment:
It certainly could be removed.  It was useful because different version of cmake have different handling for Python3 so I needed to know what was running.

I'll remove it in the next update if you like, not fussed now it builds.